pkg/libinit/net_freebsd.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2014-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build linux && !tinygo
```

Comment:
wrong build tag? file suffix `is _freebsd`

Reply:
that file was a mistake, it's gone now. Thanks.